scripts/lib/CIME/code_checker.py

```diff
@@ -19,7 +19,7 @@ def _run_pylint(on_file, interactive):
 
     cmd_options = " --disable=I,C,R,logging-not-lazy,wildcard-import,unused-wildcard-import"
     cmd_options += ",fixme,broad-except,bare-except,eval-used,exec-used,global-statement"
-    cmd_options += ",logging-format-interpolation,no-name-in-module"
+    cmd_options += ",logging-format-interpolation,no-name-in-module,arguments-renamed"
```

Comment:
will this present a problem for older pylint versions?

Reply:
I don't think so, I just checked that adding a disabling of an unknown warning is not an error.